applications/CableNetApplication/tests/cable_net_test_factory.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics as KM
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as kratos_utils
+
+
+from KratosMultiphysics.CoSimulationApplication.co_simulation_analysis import CoSimulationAnalysis
```

Comment:
this import has to be done after line 15 otherwise this will crash if CoSim is not compiled